pkg/declcfg/declcfg_to_model.go

```diff
@@ -0,0 +1,108 @@
+package declcfg
```

Comment:
There's a separate `model_to_declcfg.go`. We could combine both of these files into a `model.go` file. But I generally like to keep files in a package scoped somewhat small to make navigation and discovery easier. But I don't have any really strong feelings on this.

Reply:
I really like the break down to `model_to_declcfg.go` and `declcfg_to_model.go` instead of munging everything into one file for the same reason Joe mentioned.